_rules/inline-link-in-paragraph-is-distinguishable-be4d0c.md

```diff
@@ -0,0 +1,475 @@
+- **distinguishing content**: the [visible][] [inclusive descendant][] has adjacent content (such as an image or text) that indicates the test target is a link.
+
+## Assumptions
+
+- Any change in font is sufficiently distinguishable, and fonts are loaded when they are present.
+- The same foreground color and the same background color is used by all the visible text nodes of the [line rendering][rendered on a line] element that are not part of the target element, otherwise color can not be a distinguishing factor.
+- The same `border` is used by all the visible text nodes of the [line rendering][rendered on a line] element that are not part of the target element, otherwise `border` can not be a distinguishing factor.
+- The same `box-shadow` is used by all the visible text nodes of the [line rendering][rendered on a line] element that are not part of the target element, otherwise `box-shadow` can not be a distinguishing factor.
+- The same text style is used by all the visible text nodes of the [line rendering][rendered on a line] element that are not part of the target element, otherwise the [text style properties][] can not be a distinguishing factor.
+- The perceived visual styling of elements is the result of CSS styling applied to those elements and inherited from their [ancestor][] elements. Through manipulation of the positioning and styles of other elements, including non inherited styles of [ancestor][] elements, an element may end up with a perceived distinguishable style not resulting from its own and inherited styling. In that case, this rule might fail while the [Success Criterion 1.4.1 Use of Color](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html) might still be satisfied.
+
+## Accessibility Support
+
+_No accessibility support issues known._
+
+## Background
+
+Some examples for this rule change the default user agent styling of links. This is done to illustrate the rule. Changing the default styles of links is not endorsed by this rule.
+
+- [Understanding Success Criterion 1.4.1: Use of Color](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html)
+- [G183: Using a contrast ratio of 3:1 with surrounding text and providing additional visual cues on focus for links or controls where color alone is used to identify them](https://www.w3.org/WAI/WCAG21/Techniques/general/G183)
+- [F73: Failure of Success Criterion 1.4.1 due to creating links that are not visually evident without color vision](https://www.w3.org/WAI/WCAG21/Techniques/failures/F73)
+- [C15: Using CSS to change the presentation of a user interface component when it receives focus](https://www.w3.org/WAI/WCAG21/Techniques/css/C15)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+This link, that is distinguishable by color (hue) from the other text in the same line, uses a style similar to the default styling of links which underlines them in most browsers, making it a distinguishable style.
```

Comment:
"that is distinguishable by color" feels dangerously close to "distinguishable color" (even worse in Passed Example 5) which is about contrast, not hue… (+ link for distinguishable style)
```suggestion
This link, that has **different hue** from the other text in the same line, uses a style similar to the default styling of links which underlines them in most browsers, making it a [distinguishable style][].
```
(same for other example descriptions)